rules/windows/execution_via_compiled_html_file.toml

```
@@ -25,14 +25,68 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"
 language = "eql"
 license = "Elastic License v2"
 name = "Process Activity via Compiled HTML File"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Process Activity via Compiled HTML File
+
+CHM (Compiled HTML) files are a format for delivering online help files on Windows. CHM files are compressed compilations of various content, such as HTML documents, images, and scripting/web-related programming languages such as VBA, JScript, Java, and ActiveX.
+
+When users click those files, the HTML Help executable program (`hh.exe`) will execute them. `hh.exe` also can be used to execute code embedded in those files, PowerShell scripts, and executables, making it useful not only for attackers to proxy the execution of malicious payloads via a signed binary that could bypass security controls but to gain initial access to environments via social engineering methods.
```

Would "double-click" be more accurate? Also, trying to break up long sentence:
```suggestion
When users double-click CHM files, the HTML Help executable program (`hh.exe`) will execute them. `hh.exe` also can be used to execute code embedded in those files, PowerShell scripts, and executables. This makes it useful for attackers not only to proxy the execution of malicious payloads via a signed binary that could bypass security controls, but also to gain initial access to environments via social engineering methods.
```